CIPs/cip-0066.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
We're missing the transaction application rules (pseducode) here.

How gas is payed, how much, what happens to spread, etc. I believe some form of state transition pseudo code would be important

Reply:
Good point. I added some tx handling pseudocode.